Gemfile

```diff
@@ -44,6 +44,9 @@ gem 'omniauth-37signals'          # BasecampAgent
 # gem 'omniauth-github'
 gem 'omniauth-wunderlist', github: 'wunderlist/omniauth-wunderlist', ref: 'd0910d0396107b9302aa1bc50e74bb140990ccb8'
 
+# Uncomment to use 'em_http' as FARADAY_HTTP_BACKEND
+# gem 'em-http-request', '~> 1.1.2'
```

Comment:
Maybe add: `raise "Please edit your Gemfile and enable em-http-request" if ENV['FARADAY_HTTP_BACKEND'] == 'em_http'`

We could remove it in a few weeks once people have upgraded?